From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 19:30:31 +0200
Subject: Synchronize the provided name with the filename

The init script needs to have the same name as the thing it provides,
and at the same time it needs to have the same name as the systemd
service file, or else the /etc/init.d -> systemctl redirect won't work
correctly. (As Christoph Berg put it very succintly.)
---
 lrmd/pacemaker_remote.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lrmd/pacemaker_remote.in b/lrmd/pacemaker_remote.in
index ba89087..fd2d7b2 100644
--- a/lrmd/pacemaker_remote.in
+++ b/lrmd/pacemaker_remote.in
@@ -10,7 +10,7 @@
 # processname: pacemaker_remoted
 #
 ### BEGIN INIT INFO
-# Provides:		pacemaker_remoted
+# Provides:		pacemaker_remote
 # Required-Start:	$network
 # Should-Start:		$syslog
 # Required-Stop:	$network
